test/unit/services/dataAccessLayer.test.js

```diff
@@ -146,4 +149,53 @@ describe("Data Access Layer", function () {
       });
     });
   });
+
+  describe("privilegedAccess", function () {
+    const data = {
+      email: "test@test.com",
+      phone: "1234567890",
+      chaincode: "abc7896",
+    };
+    it("should set only email for INTERNAL access", function () {
+      const result = privilegedAccess(userData[11], data, AccessLevel.INTERNAL);
+      expect(result).to.have.property("email");
+    });
+
+    it("should set email and phone for PRIVATE access", function () {
+      const result = privilegedAccess(userData[11], data, AccessLevel.PRIVATE);
+      expect(result).to.have.property("email");
+      expect(result).to.have.property("phone");
+    });
+
+    it("should set email, phone, and chaincode for CONFIDENTIAL access", function () {
+      const result = privilegedAccess(userData[11], data, AccessLevel.CONFIDENTIAL);
+      expect(result).to.have.property("email");
+      expect(result).to.have.property("phone");
+      expect(result).to.have.property("chaincode");
+    });
+  });
+
+  describe("levelSpecificAccess", function () {
```

Comment:
why 2 different describe blocks . the previous describe block too was testing based on levels